en/flight_controller/crazyflie2.md

```diff
@@ -161,10 +163,24 @@ Then, you need to stick the battery on top of the SD card deck using a double si
 | IMU_DTERM_CUTOFF  | 70                          |
 | SYS_FMU_TASK      | Enabled                     |
 
-\* Since we haven't been able to fly it with optical flow, we don't use optical flow data yet.
+## Altitude Control 
 
+Crazyflie is able to fly in Altitude mode if you use a [Z-ranger deck](https://store.bitcraze.io/collections/decks/products/z-
+ranger-deck), connected to the board on I2C interface. The driver for this range finder is called "vl53lxx.cpp" which publishes 
+range readings on a uORB topic, named "distance_sensor". The maximum height (above ground) the range finder can sense lies in the 
+range of 0.5 to 1 m, depending on the reflective property of the ground surface. This means you cannot hold altitudes above this 
+range in Altitude or Position flight modes.
 
-### Using FrSky Taranis RC Transmitter as Joystick
+> **Tip** If in Altitude or Position mode, the crazyflie height drifts at mid-throttle command, first try rebooting the vehicle. If this does not fix the problem, recalibrate the accel and mag (compass).  
+
+> **Note** Since the onboard barometer is highly susceptible to wind disturbances created by the crazyflie's own propellers, you cannot rely on it to hold Altitude. 
+
+## Position Control 
+
+With [Flow deck](https://store.bitcraze.io/collections/decks/products/flow-deck), you can fly crazyflie in Position mode. The driver for this sensor is called "pmw3901.cpp", which publishes the flow readings on the "optical_flow" uORB topic. Unlike PX4flow, the flow deck does not house a gyro, hence the onboard gyro is used for flow fusion to find the local position estimates. Moreover, the flow deck shares the same SPI bus as the SD card deck, therefore logging at high rate on SD card is not recommended when flying in Position mode. A ulog for flight in Position mode is available [here](https://logs.px4.io/plot_app?log=a0e68bf1-e905-410f-b828-f6146dba9d45).
```

Comment:
Again, driver name is not a source file. Also, does it actually matter to users? If not, then don't tell them. 

What is the benefit of the ulog? Why are you providing this?

Reply:
I thought maybe having a ulog for our flight in position mode will serve as a reference for us and other users in the future to compare their flight performance.

Reply:
> I thought maybe having a ulog for our flight in position mode will serve as a reference for us and other users in the future to compare their flight performance.

That might be reasonable, but I'd have that in a separate note and explain why you are doing so.